cmd/agent/app/integrations.go

```diff
@@ -197,6 +198,12 @@ func tuf(args []string) error {
 	idxFlags, err := tufCmd.Flags().GetStringSlice("idx-flags")
 	if err == nil {
 		args = append(args, idxFlags...)
+	} else {
```

Comment:
I don't think this should go in the `else` here. An `err != nil` doesn't mean that the user didn't set a pip index

Reply:
That's true. More like checking for whether idxFlags is empty. Lemme update.